library/linux/Makefile

```diff
@@ -34,9 +37,16 @@ LIBRARIES=$(FLUTTER_ENGINE_LIB_FILE)
 HEADERS=$(shell find include/ -type f -name '*.h')
 SOURCES=$(shell find src/ -type f -name '*.cc')
 
-.PHONY: all
+
+# ================================== Targets ==================================
+
+.PHONY: all clean
```

Comment:
Having annotations attached directly to the target in question seems more readable to me than this version. Can you explain the motivation for this?

Reply:
I think there is no reason to disperse `PHONY` targets throughout the Makefile, so I think that my variant is more compact and readable.